internal/service/advancedclustertpf/model_ClusterDescriptionProcessArgs20240805.go

```diff
@@ -27,7 +27,7 @@ func AddAdvancedConfig(ctx context.Context, tfModel *TFModel, input *admin.Clust
 			if diags.HasError() {
 				return
 			}
-			if conversion.SafeValue(stateConfigSDK.FailIndexKeyTooLong) != conversion.SafeValue(failIndexKeyTooLong) {
+			if stateConfigSDK != nil && conversion.SafeValue(stateConfigSDK.FailIndexKeyTooLong) != conversion.SafeValue(failIndexKeyTooLong) {
```

Comment:
is threre a stateConfigSDK.GetFailIndexKeyTooLong() ?

Reply:
Great point. Changed in 9cad98866cfc8898b078e6b3ee527e458cefcef0